libraries/chain/include/scorum/chain/services/dev_pool.hpp

```diff
@@ -11,13 +11,15 @@ struct dev_pool_service_i
 {
     virtual const dev_committee_object& get() const = 0;
 
-    virtual bool is_exists() const = 0;
-
     using modifier_type = std::function<void(dev_committee_object&)>;
 
     virtual const dev_committee_object& create(const modifier_type&) = 0;
 
     virtual void update(const modifier_type&) = 0;
+
+    virtual asset get_scr_balace() const = 0;
```

Comment:
Redundantly. Using 'service.get().scr_balance' is enough.
I don't want to use getter and setter for each field of database object.